skia-bindings/build_support/skia.rs

```diff
@@ -691,7 +699,10 @@ pub fn build_skia(
     config: &BinariesConfiguration,
     ninja_command: &Path,
 ) {
+    // Libraries we explicitly want ninja to build.
+    let ninja_built_libraries = config.built_libraries.iter().filter(|x| *x != lib::SKIA_BINDINGS);
     let ninja_status = Command::new(ninja_command)
+        .args(ninja_built_libraries)
```

Comment:
Explicitly ask ninja to build the targets we want. Otherwise we end up having linker errors for skia's example programs that we don't want anyway.